cxx4opencl/diff2openclc.txt

```diff
@@ -0,0 +1,120 @@
+=== Difference to OpenCL C
```

Comment:
I've no idea if this is a requirement but other files seem to have a copyright header.